SalesforceMobileSDK-iOS.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
       smartstore.dependency 'SQLCipher', '~> 3.1'
       smartstore.dependency 'SQLCipher/fts', '~> 3.1'
-      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
+      smartstore.dependency 'FMDB', '~> 2.5'
+      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
```

Comment:
Another thing that occurs to me: have we verified that this configuration will actually encrypt the database?  Given that we're depending on a standalone version of FMDB—i.e. a separate, independent compilation unit—it seems that FMDB should be the spec responsible for including `-DSQLITE_HAS_CODEC`, not SmartStore.  I have the feeling this is not going to do what we expect it to.

FMDB does have a subspec specifically for SQLCipher: `FMDB/SQLCipher`.  However, this too could be problematic in our case, since it has a dependency on the `SQLCipher` spec, whereas we want to use our own version, since the SQLCipher spec doesn't seem to be maintained.

I'm an inch away from suggesting that we need to re-incorporate FMDB directly back into our own code base.  The maintenance issues involved with using it as a standalone library, IMO, outweigh any benefits we have from separating what's otherwise such a small and integrated code base to our SmartStore offering.

I say we keep the fork to maintain alignment, but lose the standalone build of the project.  Thoughts?